jenkins.check_copyright.sh

```diff
@@ -32,7 +32,7 @@ git diff --name-only `git merge-base origin/master HEAD` HEAD |
     grep -v -E '\.def$' |
     grep -v -E '\.inc$' |
     grep -v -E 'test/benchmarks/.*\.js$' |
-    xargs -I % sh -c "echo 'Check Copyright > Checking %'; python jenkins.check_copyright.py % > $ERRFILETEMP; if [ \$? -ne 0 ]; then cat $ERRFILETEMP >> $ERRFILE; fi"
+    xargs -I % sh -c "echo 'Check Copyright > Checking %'; python jenkins.check_copyright.py % > $ERRFILETEMP; || cat $ERRFILETEMP >> $ERRFILE; fi"
```

Comment:
@goyakin I'm not sure how difficult it would be to do but considering it is easy enough to tell if the commands fail it's probably not worth investing the time.